### ISSUE TITLE
Monthly PyLadies Meeting Notes: Jan 8 2020 @ 10:00 PST

### ISSUE THREAD

**Issue by lorenanicole** [pyladies/global-organizing#55]
_This template should be used (edited) the month prior and during the meeting. This will ensure we can address all topics added throughout the month; as well as provide minutes and follow up after the meeting is held._

**Initial Details**

- Previous meeting issue link: 
   - GitHub Issue: https://github.com/pyladies/global-organizing/issues/45
- Current meeting attendees:

**Meeting Topics**

_Please add items as needed. If you add a topic please make sure to attend the meeting._

_New Business_

- [ ] Who wants to run this meeting? Or else @lorenanicole will 😭 .
- [ ] Review PyLadies Voting #50 
     - 16 votes thus far
     - What next?
- [ ] Review PyLadies Member Database #54 
     - Anyone help with this issue?
- [ ] Review PyLadies Chapter Policies in the Project Admin & Logistics Repo 
     - How to start a chapter: https://github.com/pyladies/project-admin-logisitics/issues/8
     - Country specific chapter: https://github.com/pyladies/project-admin-logisitics/issues/9 
     - How to deactivate a chapter: https://github.com/pyladies/project-admin-logisitics/issues/10
     - How to make "unofficial" chapters official: https://github.com/pyladies/project-admin-logisitics/issues/11
- [ ] Who is attending PyCon US?
   - Yes: @jesford @deboraazevedo @lorenanicole @trallard 
- [ ]

_Old Business: Follow up items from prior meeting:_
  - [ ]
  - [ ]
  - [ ]
  - [ ]

**Action Items and Assignees for Next Meeting**

Action Item| Owner | Relevant links | 
| --| --| --|

**Comment by treasurechristain** [comment]
Please can we know the list of the chapters that voted?

**Comment by lorenanicole** [comment]
@treasurechristain posted in Slack!

**Comment by trallard** [comment]
Apologies for missing this call. I am going to PyCon US too so looking forward to seeing you all there

**Comment by Mariatta** [comment]
Did you all have chance to review the proposed policies re: starting/deactivating chapters during last meeting?

**Comment by Mariatta** [comment]
And is there any feedback?

**Comment by lorenanicole** [comment]
TLDR; no we didn't get any tangible feedback @Mariatta as we focused mainly on the election work however I did raise the issue with attendees to get their eyes on. I am late publishing the minutes from the meeting and will get that up this weekend 😱. That said, @Mariatta I don't think it hurts to do another round of outreach as well!

**Comment by lorenanicole** [comment]
Ah I just saw this @Mariatta : https://github.com/pyladies/project-admin-logisitics/pull/12/files! Looks like the @pyladies/admin-and-logistics team has started work on the form at least :-)
